src/main/java/com/schibsted/spt/data/jslt/impl/BuiltinFunctions.java

```diff
@@ -287,6 +291,30 @@ else if (!divisor.isNumber())
     }
   }
 
+  // ===== HASH-INT
+
+  public static class HashInt extends AbstractFunction {
+
+    private static ObjectMapper mapper = new ObjectMapper();
+    private static ObjectWriter writer = mapper.writerWithDefaultPrettyPrinter();
```

Comment:
We need to sort the keys, I think

Reply:
I sort the keys now: 
```
sha256-hex 2000000 iterations took 31 seconds. 64516.13 iterations/second
hash-int-v1 (string serialization)2000000 iterations took 29 seconds. 68965.516 iterations/second
hash-int-v2 (no string serialization)2000000 iterations took 27 seconds. 74074.07 iterations/second
hash-int-v3 (delegate to sha256-hex)2000000 iterations took 29 seconds. 68965.516 iterations/second
```

hash-int-v1 is still 7% slower that the hash-int-v2 .